Dockerfile

```diff
@@ -95,7 +95,7 @@ CMD ["bundle", "exec", "puma"]
 # ------------------------------------------------------------------------------
 FROM web as test
 
-RUN apt-get install -qq -y firefox-esr
+RUN apt-get update && apt-get install --fix-missing -qq -y firefox-esr
```

Comment:
What did adding `--fix-missing` fix?

Reply:
From memory (it's been a whilte since I looked at this!) the firefox install stopped working, I can only assume it was after an update to the package. The fix may have been running apt-get update first and the fix-missing was just a hangover from trying to diagnose the issue.